core/src/test/java/io/parsingdata/metal/data/SelectionTest.java

```diff
@@ -42,12 +42,21 @@
         @Override protected boolean isAvailable(BigInteger offset, BigInteger length) { return true; }
     };
 
+    private final Source otherSource = new Source() {
+        @Override protected byte[] getData(BigInteger offset, BigInteger length) { return new byte[0]; }
+        @Override protected boolean isAvailable(BigInteger offset, BigInteger length) { return true; }
+    };
```

Comment:
Minor, just wondering: is this the first time we need an "other" source? If so, maybe we can reuse it here.

Reply:
There's one other use but it's a little different. Instead I've extracted the construction into a method so that it's no longer a duplicate blob.